src/main/resources/avro/referencemethods.avdl

```diff
@@ -113,21 +113,20 @@ as JSON.
 */
 record SearchReferencesRequest {
   /**
-  If present, return only references which belong to this reference set.
+  If not null, return only references which belong to this reference set.
   */
   union { null, string } referenceSetId = null;
 
   /**
-  If nonempty, return references which match any of the given `md5checksums`.
-  See `Reference::md5checksum` for details.
+  If nonempty, return only `Reference`s on `Sequence`s with IDs in the list.
   */
-  array<string> md5checksums = [];
+  union { null, string } sequenceIds = [];
```

Comment:
This should still be array<string>, I think I mistook one object for another here. Not sure why this even builds...